llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -7265,7 +7267,7 @@ bool SPIRVToLLVM::transAddressingModel() {
 bool SPIRVToLLVM::transDecoration(SPIRVValue *bv, Value *v) {
   if (!transShaderDecoration(bv, v))
     return false;
-  m_dbgTran.transDbgInfo(bv, v);
+  m_dbgTran.recordsDbgInfo(bv, v);
```

Comment:
I don't think this fits in here since it doesn't really have to do with the value's decorations in the `OpDecorate*` sense as far as I understand it. It should be directly in `transValue`.

Actually, this entire `transDecoration` method should probably just be inlined into `transValue`. It does nothing other than forwarding the call, and so having it makes the code harder to follow.